.github/workflows/publish-grpc-libs.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish JS gRPC bindings
```

Comment:
No good way to test this, so will have to tag an alpha release once this is merged to see what happens.